schemas/compat-data.schema.json

```diff
@@ -35,25 +35,38 @@
               }
             },
             "additionalProperties": false,
-            "required": ["type", "name"]
+            "required": ["type", "name"],
+            "minProperties": 0,
+            "maxProperties": 3
           }
         },
         "partial_implementation": {
-          "type": "boolean",
+          "const": true,
           "description": "A boolean value indicating whether or not the implementation of the sub-feature deviates from the specification in a way that may cause compatibility problems. It defaults to false (no interoperability problems expected). If set to true, it is recommended that you add a note explaining how it diverges from the standard (such as that it implements an old version of the standard, for example)."
         },
         "version_added": {
           "description": "A string (indicating which browser version added this feature), the value true (indicating support added in an unknown version), the value false (indicating the feature is not supported), or the value null (indicating support is unknown).",
-          "allOf": [
-            { "type": ["string", "boolean", "null"] },
-            { "not": { "enum": ["true", "false", "null"] } }
+          "anyOf": [
+            {
+              "type": "string",
+              "pattern": "^(≤?(\\d+)(\\.\\d+)*|preview)$"
```

Comment:
Well, technically speaking, this also makes our schema "stricter", but all in all this is just enforcing what we already enforce in the schema rather than just linter files.